### PR TITLE
Remove outdated Gitter bridge reference

### PR DESCRIPTION
**Pull request by MadLittleMods**
Remove outdated [`matrix-appservice-gitter`](https://github.com/matrix-org/matrix-appservice-gitter) reference.

Don't want to confuse people on how the new Gitter bridge works as this old bridge doesn't even run anymore.

### CODE REVIEW

File: gatsby/content/docs/2017-03-11-types-of-bridging.mdx [hunk 1]
```diff
@@ -50,7 +50,7 @@ Several obstacles exist to the proper implementation of double-puppeted bridges.
 
 ### Hybrid Relaybot Puppet Bridge
 
-This type of bridge is a combination single or double puppet bridge which tries to solve the problem of representing other users by means of the bridgebot technique. [matrix-appservice-gitter](https://github.com/matrix-org/matrix-appservice-gitter) works in this way.
```
Comment (MadLittleMods) [inline gatsby/content/docs/2017-03-11-types-of-bridging.mdx:53]: Is there another bridge that works this way that we can reference here instead?

Comment (MadLittleMods) [reply gatsby/content/docs/2017-03-11-types-of-bridging.mdx:53]: It looks like https://github.com/mautrix/telegramcould fits the bill.

**Comment by MadLittleMods** [comment]
Thanks @afranke! Updated the FAQ 👍

I've left `gatsby/content/projects/2016-08-10-matrix-appservice-gitter-twisted.mdx` alone as I don't know what that authors plans are (if they actually want to deprecate in favor of the native bridge). It's already marked as `maturity: Not actively maintained` and not `featured` so it doesn't show up. I've never even heard of it for example.

**Comment by afranke** [comment]
Fair enough.

**Pull request merged**